Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
Yeah, doesn't need a new bottle, will only do "unexpected things" for users building from source rather than from bottle. Thanks for the fix!